### PR TITLE
 Makes photocopiers less exploitable.

### LINKED ISSUE #6544: Photocopiers are exploitable
With photocopiers, you can:
Climb on it with no delay, or move someone else on it with no delay

Completely ignore projectiles like lasers while sitting on it

Stack multiple mobs by dragging them all onto it and they wont push each other off

Wrench and unwrench it instantly (not a big issue since you can climb onto them to get around them, but can still block people from pulling objects that can't pass over a photocopier)

And my personal favorite: You can push a photocopier underneath plastic flaps then climb onto the photocopier to pass through the flaps.

### PR DESCRIPTION
**Pull request by theOperand**
- Adds a delay to (un)wrenching and putting people on photocopiers.
- You can no longer put people on photocopiers if there are other dense things on the same tile.

Also makes the proc for putting people on copiers less awful.

Fixes #6544.

Tested.

### CODE REVIEW

File: code/modules/paperwork/photocopier.dm [hunk 1]
```diff
@@ -333,6 +339,16 @@
 	else
 		return 1
 
+/obj/machinery/photocopier/proc/copier_blocked()
+	if(gc_destroyed)
+		return
+	for(var/atom/movable/AM in loc)
+		if(AM == src)
+			continue
+		if(AM.density || AM.density)
```
Comment (Menshin) [inline code/modules/paperwork/photocopier.dm]: Honk

Comment (Miauw62) [inline code/modules/paperwork/photocopier.dm]: welp.

**Comment by Ergovisavi** [comment]
This does not appear to work, attempting to wrech/unwrench a photocopier gives you the "You start wrenching the photocopier" text, but nothing actually comes of it.

**Comment by Miauw62** [comment]
are you sure?
because it should work...

**Comment by Ergovisavi** [comment]
Tested again, wrenching runtime has been fixed, but you get the message "You start wrenching the photocopier" for unanchoring it, and "You start unwrenching the photocopier" for anchoring it, these messages are reversed from what they should be.

 In addition, photocopier can still be used to phase through walls by climbing on top of it, assuming that it was built on a wall. Please fix.

**Comment by Miauw62** [comment]
can photocopiers be built?
if a photocopier can end up on a wall, it's either raz's fault or whoever wrote wall construction code.

**Comment by Miauw62** [comment]
anyway, fixt.
also added a check against fastening it in space.

**Comment by Ergovisavi** [comment]
Tested, everything appears to be in order.

**Pull request merged**